app/controllers/users_controller.rb

```diff
@@ -34,7 +34,7 @@ def new_with_invitation_token
   private
     
   def user_params
-    params.require(:user).permit(:email, :password, :full_name)
+    params.require(:user).permit(:email, :password, :full_name, :admin)
```

Comment:
If you add `:admin` here, that means user can change it himself if he manages to send that via HTML or a POST request.